src/Pmi/Controller/AdminController.php

```diff
@@ -425,66 +420,6 @@ public function missingOrdersAction(Application $app, Request $request, $_route)
         ]);
     }
 
-    public function noticesAction(Application $app)
-    {
-        $notices = $app['em']->getRepository('notices')->fetchBy([], ['id' => 'asc']);
-        return $app['twig']->render('admin/notices/index.html.twig', [
-            'notices' => $notices
-        ]);
-    }
-
-    public function noticeAction($id, Application $app, Request $request)
-    {
-        if ($id) {
-            $notice = $app['em']->getRepository('notices')->fetchOneBy(['id' => $id]);
-            if (!$notice) {
-                $app->abort(404);;
-            }
-        } else {
-            $notice = null;
-        }
-
-        $form = $app['form.factory']->createNamed(
-            'form',
-            NoticeType::class,
-            $notice ?: ['status' => 1],
```

Comment:
This functionality didn't make it to the new controller. When creating a new notice, the enabled status should default to true. (i.e. create a new Notice instance and set the status instead of passing null to the form here: https://github.com/all-of-us/healthpro/pull/703/files#diff-0131fcea65cb24fc2f156786d74ade40R42)

Reply:
Resolved in d8bb5a2.